### DISCUSSION TITLE
status of ubus support in otbr-agent

### DISCUSSION THREAD

**Discussion by jascrain** [openthread/openthread#9124] [Q&A] [answered]
I am looking into running otbr-agent on a wifi router. I'd prefer to avoid using ot-ctl to communicate with the agent, and I see that it has a ubus interface.

I don't see a lot of churn in otbr's ubus code. Is the ubus interface a good / recommended way to manage otbr-agent?

If I find that the ubus interface is missing some functionality, are you willing to review a pull request?

**Comment by jwhui** [comment, marked as answer]
I'm not aware of anyone that is actively driving/maintaining the ubus implementation at this time. As an open-source project, we would welcome contributions to help carry it forward!
